client/src/App.js

```diff
@@ -98,7 +99,9 @@ class App extends Component {
                 first_name: json.profile.first_name,
                 image: json.profile.image,
                 userid: json.id,
-                profileLoaded: true
+                profileLoaded: true,
+                connections: json.profile.connections,
```

Comment:
Can you move this ajax request into a service? All ajax requests should be an a service in the `services/` directory